src/ccl_correlation.c

```diff
@@ -506,3 +509,419 @@ void ccl_correlation_3d(ccl_cosmology *cosmo, double a,
 
   return;
 }
+
+/*--------ROUTINE: ccl_correlation_multipole ------
+TASK: Calculate multipole of the redshift space correlation function. Do so using FFTLog.
+
+INPUT:  cosmology, scale factor a, beta (= growth rate / bias), 
+        multipole order l = 0, 2, or 4, number of s values, s values
+
+Multipole function result will be in array xi
+ */
+
+void ccl_correlation_multipole(ccl_cosmology *cosmo, double a, double beta,
+                               int l, int n_s, double *s, double *xi,
+                               int *status) {
+  int i, N_ARR;
```

Comment:
Shouldn't this check if I have a splined version and if so use that?

Reply:
Good point. We will do that.

Reply:
It is not necessary. We only create the global spline (ccl_correlation_multipole_spline) since ccl_correlation_pi_sigma needs to call this many times.  ccl_correlation_multipole and  ccl_correlation_multipole_spline are not needed by the user, they are only used to calculate xi(s,mu), xi(pi,sigma) and xi(s).